docs/integration/bindings/micropython.rst

```diff
@@ -182,7 +182,7 @@ The Micropython Binding is auto generated!
 LVGL C API Coding Conventions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For a summary of coding conventions to follow see the `CODING STYLE <CODING_STYLE>`__.
+For a summary of coding conventions to follow see the `CODING STYLE <../../CODING_STYLE.html>`__.
```

Comment:
Would it be better to use the `ref` feature of rst? Not sure what it's actually called. Example: [ref label](https://github.com/lvgl/lvgl/blob/b44bcaee17b7ed37fd808a86fc9cd0c91e851e28/docs/get-started/quick-overview.rst?plain=1#L1) and [where the label is referenced](https://github.com/lvgl/lvgl/blob/b44bcaee17b7ed37fd808a86fc9cd0c91e851e28/docs/get-started/index.rst?plain=1#L13). Maybe it's fine as-is.

Reply:
Good idea!

Reply:
Definitely better. I'm new to Sphinx, although suspected such feature exists.